x-pack/metricbeat/module/aws/s3/_meta/fields.yml

```diff
@@ -0,0 +1,14 @@
+- name: s3
+  type: group
+  description: >
+    `s3` contains the metrics that were scraped from AWS CloudWatch which contains monitoring metrics sent by AWS S3.
+  release: beta
+  fields:
+    - name: bucket.size.bytes
```

Comment:
I see additional fields in the data.json above then just these?

Also why is this float? I would assume this should be a long?

Reply:
Yes I missed bucket.name and bucket.storage.type. Thanks! It's float because `bucket.size.bytes` is also an average value cloudwatch gives.

Reply:
I wonder if we should indicate that somehow in the name as I would expect this to not be an average. So this is the average size over the last 24 hours? I wonder how this is calculated :-)

Reply:
Hmm maybe changing the metric name to `bucket.size.avg.bytes`?  I would like to keep the decimals even though they are not very useful but at least we are keeping whatever we are getting from AWS :-)